packages/formation-react/src/components/ResponsiveTable/ResponsiveTable.stories.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+
+import ResponsiveTable from './ResponsiveTable';
+
+export default {
+  title: 'Library/ResponsiveTable',
+  component: ResponsiveTable,
+};
```

Comment:
Can you add controls for `currentSort.order` to make it an enum selection of `ASC` and `DESC`? (Not sure if you can do nested controls like that, though. I'd prefer two separate props, but...we're just copying the component, so :see_no_evil:. Bonus points if you can figure out better controls for `fields`.)

Reply:
I've poked around a bit and can't seem to find a way to have nested controls like that. :upside_down_face: